heroku/deploy-a-django-app.md

```diff
@@ -299,45 +299,29 @@ apps](https://devcenter.heroku.com/articles/github-integration-review-apps)
 for your pipeline:
 
 ```bash
-# If you only have a staging app, change the value of the -a flag to ${APP_NAME}-staging
-# to set the staging app as the parent app for all review apps.
-heroku reviewapps:enable -p ${APP_NAME} -a ${APP_NAME}
+# Note that these need to be two separate commands due to an open Heroku bug,
+# since --autodeploy and --autodestroy require a PATCH request
+heroku reviewapps:enable -p ${APP_NAME}
+heroku reviewapps:enable -p ${APP_NAME} --autodeploy --autodestroy
 ```
 
-Next, configure environment variables for staging, production, and review apps. These can be set
-using either the dashboard or the CLI. [Follow the Heroku
-documentation](https://devcenter.heroku.com/articles/config-vars#managing-config-vars)
-to set up your config vars. Note that you need to define config vars for each environment
-in your application: for staging and production environments, you can define these
-variables under `Settings > Config Vars` on the application page, but since review apps
-don't have an application page until they're created, you can define config vars
-that will be inherited by each review app by navigating to the pipeline home page
-and visiting `Settings > Review Apps > Review app config vars` in the nav.
-
-If you followed the setup instructions in "[Read settings and secret variables from the
-environment](#read-settings-and-secret-variables-from-the-environment)" above,
-you should need to set at least the following variables:
-
-- `DJANGO_SECRET_KEY`: This needs to be a random string, unique to each environment.
-  We often use the [XKCD password generator](https://preshing.com/20110811/xkcd-password-generator/)
-  to generate random strings.
-- `DJANGO_ALLOWED_HOSTS`: This variable should be a comma-separated list of hostnames
-  that are valid for your application. For review apps and the staging environment,
-  you can set this to `.herokuapp.com` to automatically safelist any subdomains
-  of the `herokuapp.com` root domain.
-
-Note that while `DATABASE_URL` is probably required by your application, you don't actually
-need to set it yourself. The Heroku Postgres add-on will [automatically define this
-variable](https://devcenter.heroku.com/articles/heroku-postgresql#designating-a-primary-database)
-when it provisions a database for your application.
```

Comment:
Might be nice to keep this note.

Reply:
Good thinking, added it back in e50a7d4!